src/dmd/inline.d

```diff
@@ -1800,7 +1872,7 @@ Lno:
  */
 private void expandInline(Loc callLoc, FuncDeclaration fd, FuncDeclaration parent, Expression eret,
         Expression ethis, Expressions* arguments, bool asStatements,
-        out Expression eresult, out Statement sresult, out bool again)
+        out Expression eresult, out Statement sresult, out bool again, VarDeclaration vthis2)
```

Comment:
Please move vthis2 before all the `out` parameters

Reply:
done